### PR TITLE
magento/merchdocs#1126: Add the "Reset" button clarification to the Grid Layout page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds clarification to the "Reset" button on the grid layout page.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/stores/admin-grid-layout.html

## Additional information

Grid Layout | Magento 2 4 User Guide 2021-04-02 14-51-20

### CODE REVIEW

**Comment by atwixfirster** [comment]
Notice makes sense.

Thank you for your contributions, @ZhovnirMaria

File: src/stores/admin-grid-layout.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ _Order Grid Columns_
 
    - Select the checkbox of any column you want to add to the grid.
    - Clear the checkbox of any column you want to remove from the grid.
+   - Click the **Reset** button to return the default grid view.
```
Comment (leslietilling) [inline src/stores/admin-grid-layout.md:19]: style tweak

```suggestion
   - Click **Reset** to return the default grid view.
```

**Comment by ZhovnirMaria** [comment]
@leslietilling 
Done✅ Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1127`.

**Pull request merged**